modules/framework/galasa-parent/dev.galasa.framework.resource.management/src/test/java/dev/galasa/framework/resource/management/internal/TestDssEventWatcher.java

```diff
@@ -20,38 +21,38 @@ public class TestDssEventWatcher {
 
     @Test
     public void testNullKeyGetsIgnored() {
-        BlockingQueue<DssEvent> eventQueue = null;
+        Queue<DssEvent> eventQueue = null;
         IDynamicStatusStoreService dss = null;
         DssEventWatcher watcher = new DssEventWatcher(eventQueue,dss);
         watcher.propertyModified(null , null , null , null);  
     }
 
     @Test
     public void testNullEventGetsIgnored() {
-        BlockingQueue<DssEvent> eventQueue = null;
+        Queue<DssEvent> eventQueue = null;
         IDynamicStatusStoreService dss = null;
         DssEventWatcher watcher = new DssEventWatcher(eventQueue,dss);
         watcher.propertyModified( "something" , null , null , null);  
     }
 
     @Test
     public void testIgnoresDssModifyIfThereIsNoRunId() {
-        BlockingQueue<DssEvent> eventQueue = null;
+        Queue<DssEvent> eventQueue = null;
         IDynamicStatusStoreService dss = null;
         DssEventWatcher watcher = new DssEventWatcher(eventQueue,dss);
         watcher.propertyModified( "somethingWhichFailsTheRegexWithNoRunId" , Event.MODIFIED , "old" , "new");  
     }
 
     @Test
     public void testDssModifyWithARunIdEnqueuesAnEvent() throws Exception {
-        BlockingQueue<DssEvent> eventQueue = new LinkedBlockingQueue<DssEvent>();
+        Queue<DssEvent> eventQueue = new LinkedBlockingDeque<DssEvent>();
```

Comment:
Should this be `LinkedBlockingQueue` instead of a Deque? Same applies for the other tests

Reply:
Yep. Though they did work, oddly.